tools/frr

```diff
@@ -131,7 +131,7 @@ start()
               if [ -e /var/run/frr/watchfrr.started ] ; then
 		  break
               else
-                  sleep 1
+                  sleep 0.5
```

Comment:
Shouldn't we double the "for i in `seq 1 10`;" just above here?

Reply:
This `sleep` call does not exist anymore after #3098